eatin-client/src/homePage/HomePage.tsx

```diff
@@ -1,5 +1,176 @@
+    const cakes = useGetTopRatedRecipesByCategoryQuery({ variables: { category: Category.Cake } })
+        .data?.topRecipesByCategory;
+    const japanese = useGetTopRatedRecipesByCuisineQuery({
+        variables: { cuisine: Cuisine.Japanese },
+    }).data?.topRecipesByCuisine;
+    const greek = useGetTopRatedRecipesByCuisineQuery({ variables: { cuisine: Cuisine.Greek } })
+        .data?.topRecipesByCuisine;
+
+    // This is temp
+    function getItemsArray(recipes: any) {
+        const items: Item[] = [];
+        recipes?.forEach(
+            (recipe: {
+                index: string;
+                recipe_title: string;
+                url: string;
+                record_health: string;
+                vote_count: string;
+                rating: string;
+                description: string;
+                cuisine: string;
+                course: string;
+                diet: string;
+                prep_time: string;
+                cook_time: string;
+                ingredients: string;
+                instructions: string;
+                author: string;
+                tags: string;
+                category: string;
+                image: string;
+                difficulty: string;
+            }) =>
+                items.push(
+                    new Item(
+                        recipe.index,
+                        recipe.recipe_title,
+                        recipe.url,
+                        recipe.record_health,
+                        recipe.vote_count,
+                        recipe.rating,
+                        recipe.description,
+                        recipe.cuisine,
+                        recipe.course,
+                        recipe.diet,
+                        recipe.prep_time,
+                        recipe.cook_time,
+                        recipe.ingredients,
+                        recipe.instructions,
+                        recipe.author,
+                        recipe.tags,
+                        recipe.category,
+                        recipe.image,
+                        recipe.difficulty,
+                    ),
+                ),
+        );
+        return items;
+    }
+
+    // This is temp
+    useEffect(() => {
+        const eggItems = getItemsArray(eggs);
+        const cakeItems = getItemsArray(cakes);
+        const japaneseItems = getItemsArray(japanese);
+        const greekItems = getItemsArray(greek);
+        setCurrentShownRecipes([
+            { name: Category.Egg.toString(), items: eggItems },
+            { name: Category.Cake.toString(), items: cakeItems },
+            { name: Cuisine.Japanese.toString(), items: japaneseItems },
+            { name: Cuisine.Greek.toString(), items: greekItems },
+        ]);
+    }, [currentShownRecipes, cakes, eggs, japanese, greek]);
+
+    const currentFilterOptions: {
+        name: string;
+        options: string[];
+        funcToUpdate: (arg0: string) => void;
+    }[] = [
+        {
```

Comment:
do this as a query from the backend, or if not now for the prototype - add a `TODO` comment and open an issue in the board